packages/plugin-mongo/src/database.ts

```diff
@@ -43,4 +49,17 @@ export default class MongoDatabase {
   stop() {
     return this.client.close()
   }
+
+  connectionStringFromConfig() {
+    let mongourl = `${this.config.protocol}://`
+    if (this.config.username) mongourl += `${this.config.username}${this.config.password ? `:${this.config.password}` : ''}@`
+    mongourl += `${this.config.host}${this.config.port ? `:${this.config.port}` : ''}/${this.config.authDatabase ? this.config.authDatabase : this.config.name}`
+    if (this.config.connectionOption) {
+      // https://nodejs.org/api/url.html#url_new_urlsearchparams_obj this should be find but I got an complaint from TS
+      const params = new URLSearchParams(this.config.connectionOption)
+      params.sort()
```

Comment:
why it must be sorted?

Reply:
oh my bad it doesn't

Reply:
02ff7d9caa6430baf9c6f979dd30042e9c922179 fixed